lib/normalizy/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Normalizy
-  VERSION = '1.3.0'
```

Comment:
Usually, the maintainer changes it, since we don't know when the patch will be applied.